src/platform/user/profile/vet360/components/base/Vet360TransactionErrorBanner.jsx

```diff
@@ -21,11 +25,70 @@ export function GenericUpdateError(props) {
   );
 }
 
+export function VAProfileInitError(props) {
+  return (
+    <AlertBox
+      {...props}
+      className="vads-u-margin-top--0 vads-u-margin-bottom--4"
+      status="error"
+      headline="We can’t load some of your information"
+      content={
+        <div>
+          <p>
+            We’re sorry. We can’t load some of the information in your profile.
+            This may be because you have multiple IDs or accounts at VA.
+          </p>
+          <p>
+            <strong>
+              To find out if this about an account on My HealtheVet:
+            </strong>
+          </p>
+          <p>
+            Contact us at <Telephone contact={CONTACTS.MY_HEALTHEVET} />. We’re
+            here Monday - Friday, 8:00 a.m - 8:00 p.m. ET. If you have hearing
+            loss, call <Telephone contact="800-877-3399" />. Tell the
+            representative that you tried to sign in to VA.gov, but got an error
+            message that you may have more than one My HealtheVet account or ID.
+          </p>
+          <p>
+            Or, you can{' '}
+            <a
+              href="https://www.myhealth.va.gov/mhv-portal-web/contact-us"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              fill out a My HealtheVet online help form
+            </a>{' '}
```

Comment:
Did you mean to add those spaces? `{' '}`

Reply:
Yeah that's Prettier doing its thing with long text in JSX.